app/assets/javascripts/modules/track-qa-choices.js

```diff
@@ -0,0 +1,83 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (global, GOVUK) {
+  'use strict'
+
+  GOVUK.Modules.TrackQaChoices = function () {
+    this.start = function (element) {
+      track(element[0])
+    }
+
+    function getQuestionKey(submittedForm) {
+      return submittedForm.getAttribute('data-question-key');
+    }
+
+    function triggerEventForMultipleChoice(submittedForm) {
+      var checkedOptions = submittedForm.querySelectorAll('input:checked')
+      var questionKey = getQuestionKey(submittedForm)
+
+      if (checkedOptions.length) {
+        checkedOptions.forEach(function (checkedOption) {
+          var checkedOptionId = checkedOption.getAttribute('id')
+          var checkedOptionLabel = document.querySelectorAll('label[for="' + checkedOptionId + '"]')[0].innerText.trim()
+          var eventLabel = checkedOptionLabel.length
+            ? checkedOptionLabel
+            : checkedOption.value
+
+          var options = { transport: 'beacon', label: eventLabel }
+
+          GOVUK.analytics.trackEvent('question_answer', questionKey, options)
+        })
+      } else {
+        // Skipped questions
+        var options = { transport: 'beacon', label: 'no choice' }
```

Comment:
Nit:
```suggestion
        var options = { transport: 'beacon', label: 'no response' }
```
Just to keep terminology consistent